package.json

```diff
@@ -4,6 +4,9 @@
     "description": "Hazelcast - open source In-Memory Data Grid - client for Node.js",
     "main": "lib/index.js",
     "types": "lib/index.d.ts",
+    "files": [
+        "lib/"
```

Comment:
what about README and LICENSE?

Reply:
They are included by default, check this link in the linked issue https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files